main.js

```diff
@@ -6,3 +6,37 @@ JS:
 4. bonus: clicking on the translucent background closes the dialog
 5. bonus: use Class
 */
+let str = "{{socialNetworkName}}";
+let socialNetworkName = '';
+
+function toggleDialog() {
+  $('.dialog').toggleClass('is-hidden');
+}
+
+function replaceNetworkName(name) {
+  const regex = new RegExp(`${str}`, 'g')
+  const newhtml = $('.question').html().replace(regex, name);
+  $('.question').html(newhtml);
+  str = name;
+}
+
+$('.social-link').on('click', function() {
+  socialNetworkName = $(this).attr('data-url');
+  toggleDialog();
+  replaceNetworkName(socialNetworkName);
+});
+
+$('.button--ok').on('click', function() {
+	//seems that redirection doesn't work on jsfiddle, but the function works correctly
```

Comment:
There is an error message 
`Mixed Content: The page at 'https://jsfiddle.net/marwinn/1v3x58jo/6/' was loaded over HTTPS, but requested an insecure frame 'http://facebook.com/'. This request has been blocked; the content must be served over HTTPS.`

You should try window.open instead for example